glusterd2/commands/volumes/volume-create.go

```diff
@@ -17,6 +17,20 @@ import (
 	"github.com/pborman/uuid"
 )
 
+
+// VolCreateRequest defines the parameters for creating a volume in the volume-create command
+type VolCreateRequest struct {
+	Name         string            `json:"name"`
+	Transport    string            `json:"transport,omitempty"`
+	ReplicaCount int               `json:"replica,omitempty"`
+	Bricks       []string          `json:"bricks,omitempty"`
+	Force        bool              `json:"force,omitempty"`
+	Options      map[string]string `json:"options,omitempty"`
+	Size         int               `json:"size,omitempty"`
```

Comment:
Does this field represent the targetted size of volume like say 30GB or something?

Reply:
Yes, instead of mentioning the bricks we can directly provide size and then the bricks will be allocated.